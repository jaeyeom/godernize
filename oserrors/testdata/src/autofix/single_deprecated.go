package autofix

import (
	"fmt"
	"os"
)

var _ = singleDeprecated

func singleDeprecated() {
	var err error
	if os.IsNotExist(err) { // want `os.IsNotExist is deprecated, use errors.Is\(err, fs.ErrNotExist\) instead`
		fmt.Println("File does not exist")
	}
}
