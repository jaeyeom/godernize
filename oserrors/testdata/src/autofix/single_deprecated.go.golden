package autofix

import (
	"fmt"
	"os"
)

var _ = singleDeprecated

func singleDeprecated() {
	var err error
	if errors.Is(err, fs.ErrNotExist) { // want `os.IsNotExist is deprecated, use errors.Is\(err, fs.ErrNotExist\) instead`
		fmt.Println("File does not exist")
	}
}